package.json

```diff
@@ -70,7 +70,7 @@
   },
   "scripts": {
     "start": "craco start",
-    "mock": "REACT_APP_MOCK=true && yarn start",
+    "mock": "REACT_APP_MOCK=true yarn start",
```

Comment:
Mulig at mockingen ikke fungerer lengre for Bendik på Windows med denne endringen

Reply:
@bendiksi hvis det ikke funker for deg å kjøre appen så kan det være pga dette